packages/compiler/src/experimental/typekit/kits/type.ts

```diff
@@ -33,7 +33,7 @@ export interface TypeTypekit {
    */
   finishType(type: Type): void;
   /**
-   * Checks if a type is decorated with @error
+   * Checks if a type is decorated with \@error
    * @param type The type to check.
    */
   isError(type: Type): boolean;
```

Comment:
```suggestion
  isError(type: Type) assets type is Model;
```

coudl also maybe do that to get a more precise type

Reply:
Yup, although just to clarify I assume we want the typeguard `type is Model` , not the type-assertion (`asserts type is Model`) here right? It should be safe to call with non-error types? Gonna assume yes and resolve but feel free to let me know if you meant something else!